distribution/src/config/elasticsearch.yml

```diff
@@ -80,9 +80,3 @@ ${path.logs}
 #gateway.recover_after_nodes: 3
 #
 # For more information, consult the gateway module documentation.
-#
-# ---------------------------------- Various -----------------------------------
-#
-# Require explicit names when deleting indices:
-#
-#action.destructive_requires_name: true
```

Comment:
Since the setting is now set to true by default, I think that this isn't needed anymore.

Reply:
I'm not 100% sure, I'd rather have someone else to double check this.

Reply:
I recommend leaving this block, but changing the description to "allow wildcard deletion of indices" and changing the value from "true" to "false."

Reply:
Done.